.tools/test/stacks/nuke/typescript/account_nuker.ts

```diff
@@ -0,0 +1,667 @@
+        type: 'NO_SOURCE',
+      },
+    });
+
+    const nukeStepFunctionRole = new iam.CfnRole(this, 'NukeStepFunctionRole', {
+      roleName: 'account-nuker-codebuild-state-machine-role',
+      assumeRolePolicyDocument: {
+        Version: '2012-10-17',
+        Statement: [
+          {
+            Effect: 'Allow',
+            Principal: {
+              Service: [
+                `states.${this.region}.amazonaws.com`,
+              ],
+            },
+            Action: [
+              'sts:AssumeRole',
+            ],
+          },
+        ],
+      },
+      tags: [
+        {
+          key: 'DoNotNuke',
+          value: 'True',
+        },
+        {
+          key: 'owner',
+          value: props.owner!,
+        },
+      ],
+      path: '/',
+      policies: [
+        {
+          policyName: 'account-nuker-codebuild-state-machine-policy',
+          policyDocument: {
+            Version: '2012-10-17',
+            Statement: [
+              {
+                Effect: 'Allow',
+                Action: [
+                  'codebuild:StartBuild',
+                  'codebuild:StartBuild',
+                  'codebuild:StopBuild',
+                  'codebuild:StartBuildBatch',
+                  'codebuild:StopBuildBatch',
+                  'codebuild:RetryBuild',
+                  'codebuild:RetryBuildBatch',
+                  'codebuild:BatchGet*',
+                  'codebuild:GetResourcePolicy',
+                  'codebuild:DescribeTestCases',
+                  'codebuild:DescribeCodeCoverages',
+                  'codebuild:List*',
+                ],
+                Resource: [
+                  nukeCodeBuildProject.attrArn,
+                ],
+              },
+              {
+                Effect: 'Allow',
+                Action: [
+                  'events:PutTargets',
+                  'events:PutRule',
+                  'events:DescribeRule',
+                ],
+                Resource: `arn:aws:events:${this.region}:${this.account}:rule/StepFunctionsGetEventForCodeBuildStartBuildRule`,
+              },
+              {
+                Effect: 'Allow',
+                Action: [
+                  'states:DescribeStateMachine',
+                  'states:ListExecutions',
+                  'states:StartExecution',
+                  'states:StopExecution',
+                  'states:DescribeExecution',
+                ],
+                Resource: [
+                  `arn:aws:states:${this.region}:${this.account}:stateMachine:account-nuker-codebuild-state-machine`,
+                ],
+              },
+            ],
+          },
+        },
+      ],
+    });
+
+    const nukeStepFunction = new stepfunctions.CfnStateMachine(this, 'NukeStepFunction', {
+      stateMachineName: 'account-nuker-codebuild-state-machine',
+      roleArn: nukeStepFunctionRole.attrArn,
+      definitionString: `{
```

Comment:
There's a lot of hardcoded documents in here. Can we extract them into their own files, and include them with [`readFileSync`](https://nodejs.org/api/fs.html) from `node:fs`?

Reply:
I can certainly do it for this larger document, though I'll also have to add in a line or two of code to modify the variable for `ProjectName` (which I don't love).  I can also extract the value for `source` on L367 into a file.